src/framework/preferences/messages/linksControls.json

```diff
@@ -1,3 +1,4 @@
 {
-    "linksControlsLabel": "Links & buttons"
+    "emphasisDescr": "Emphasize input options",
```

Comment:
Unofficially, they were in the design mock for suggested UIO updates https://www.dropbox.com/s/02j7ilnhxd7c8hv/UIO_mock.pdf?dl=0